core/domain/activity_domain.py

```diff
@@ -28,17 +28,30 @@ class ActivityReference(object):
     """
```

Comment:
There's a difference. The attributes are the "self.x" stuff. The method docstrings are for the arguments to `__init__`.